frontend/src/routes/routes.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Route, Switch } from "react-router-dom";
+
+import Frontpage from "../pages/Frontpage";
+import CreatePost from "../pages/CreatePost";
+import PostDetail from "../pages/PostDetail";
+
+const paths = {
+  FRONTPAGE: "/",
+  CREATE_POST: "/new",
+  POST_DETAILS: "/post"
```

Comment:
An example of airbnb's style guide is that it'll require trailing commas on this line.

Two reasons:
1. Reduce copy and pasting error
2. Cleaner git diffs when adding a new element to the end, or reordering them.

```suggestion
  POST_DETAILS: "/post",
```

This is just an opinion - I'm happy for you to disagree to it or if you feel like it's not worth the effort. :)

Reply:
Trailing commas produce linting issues. If everyone is happy to allow trailing commas, I will disable the rule.

Reply:
Ah I see - it's Prettier's formatting rules. I'm happy to stick to Prettier's style instead of airbnb's style if others are happy with it.

Reply:
@ErnWong  are you happy to merge for now